sal_interfaces/PMD/PMD_Events.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <SALEvent>
+    <Subsystem>PMD</Subsystem>
+    <EFDB_Topic>PMD_logevent_metadata</EFDB_Topic>
+    <item>
+      <EFDB_Name>kind</EFDB_Name>
```

Comment:
I'd prefer "type" but I imagine you avoided that word on purpose?
maybe "instrument" or "instrument_function" ?
I imagine you don't want to use just "function"?

Reply:
In the CSC code you have hub_type which seems to serve the same purpose. As I said in a general comment I just added perhaps hubType as an int here and add an enumeration (with just one value at the moment) would do the job. I suspect you could get rid of "kind".

Reply:
I renamed kind to hubType and added the enumeration as suggested by @r-owen.